lib/govuk_design_system_formbuilder/traits/conditional.rb

```diff
@@ -8,8 +8,11 @@ def conditional_id
       end
 
       def wrap_conditional(block)
+        block_result = block.call
+        return if block_result.nil? || block_result.eql?('')
```

Comment:
Minor one but how about...
```suggestion
        return if block_result.blank?
```
or...
```suggestion
        return unless block_result.present?
```

Reply:
Thanks @cpjmcquillan. Does the form builder aim to be framework (rails/activesupport) agnostic? I had not used `blank?` or `present?` to avoid those dependencies but would use `blank?` otherwise.

Reply:
I see it is already using `blank?` in other place actually so will tweak as above.